localdev/configs/ocw-course-site-config.yml

```diff
@@ -25,6 +25,23 @@ collections:
           - { label: "City", name: "city", widget: "string", required: true }
           - { label: "State", name: "state", widget: "string", required: true }
           - { label: "Zip Code", name: "zip", widget: "string", required: true }
+      - {
+        "label": "Authors",
+        "name": "authors",
+        "widget": "relation",
+        "collection": "author",
+        "display_field": "title",
+        "multiple": true,
+        "max": 10,
+        "min": 1,
+        }
```

Comment:
Does this need to be a required field? I haven't seen author data in the front matter in any of the markdown pages for existing OCW course sites. If it doesn't need to be required, let's add `"required": false` here. If you want to do the same for the `address` field above while you're at it, I wouldn't complain 😁